package.json

```diff
@@ -140,5 +140,8 @@
     "webpack": "5.50.0",
     "webpack-bundle-analyzer": "4.4.2",
     "webpack-cli": "4.7.2"
+  },
+  "dependencies": {
+    "morgan": "1.10.0"
```

Comment:
@charellesandig I still think this one is not needed. I saw your other edits to the child package.json files and that looks good. This here can be removed entirely IMO.

Reply:
Hi @petermetz, I deleted the 2 files you mentioned and just keep the child package.json and yarn.lock files. And is now ready for review again, thank you.

Reply:
@charellesandig Thank you!